libcontainer/configs/validate/rootless.go

```diff
@@ -51,9 +51,8 @@ func rootlessEUIDMappings(config *configs.Config) error {
 	return nil
 }
 
-// mount verifies that the user isn't trying to set up any mounts they don't have
-// the rights to do. In addition, it makes sure that no mount has a `uid=` or
-// `gid=` option that doesn't resolve to root.
+// rootlessEUIDMount verifies that all mounts have valid uid=/gid= options,
```

Comment:
This is slightly incorrect. Should have been something like

```suggestion
// rootlessEUIDMount verifies that all mounts' uid=/gid= options, if present, are valid,
```